docs/root/start/sandboxes/opentelemetry.rst

```diff
@@ -112,6 +113,81 @@ In the ``Latency Samples`` of ``opentelemetry.proto.collector.trace.v1.TraceServ
 
 .. image:: /start/sandboxes/_static/opentelemetry-ui-traces.png
 
+Step 5: View the access logs in the OpenTelemetry collector output
```

Comment:
could you swap step 4 and step 5 please - i think the screenshots should be the last step

Reply:
Done in b4a96fe0f5875d108a49b54fa30720273c35dcb8